eyelid-match-ops/src/primitives/yashe/test/hom.rs

```diff
@@ -84,16 +111,22 @@ fn homomorphic_addition_test() {
     // TODO: get TinyTest working in this module
     homomorphic_addition_helper::<MiddleRes>();
     homomorphic_addition_helper::<FullRes>();
+    homomorphic_addition_helper::<LargeRes>();
```

Comment:
We might also want a separate ignored test for all the `LargeRes` tests in this file.

We can run them in CI in a separate job?

Reply:
`LargeRes` tests is not taking so long.